config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -164,7 +164,7 @@ periodics:
           - --env=CONTAINER_IMAGE=registry-sandbox.k8s.io/pause:3.1 #TODO(ameukam): revert when registry.k8s.io is ready
           - --extract=ci/latest
           - --gcp-node-image=gci
-          - --gcp-master-size=e2-standard-32
+          - --gcp-master-size=n1-standard-32
```

Comment:
Sorry - I missed one more thing - can you please add:
```
--gcp-project-type=scalability-project
```

in both tests?

Reply:
sure, what does it do :) ?

Reply:
pushed

Reply:
It's choosing a separate pool of projects that has a bit higher quotas and simplify cost attribution to scalability-related stuff.